app/src/ui/changes/changes-list.tsx

```diff
@@ -57,6 +56,30 @@ const StashIcon = new OcticonSymbol(
 
 const GitIgnoreFileName = '.gitignore'
 
+/** Compute the 'Include All' checkbox value from the repository state */
+function getIncludeAllValue(
+  workingDirectory: WorkingDirectoryStatus,
+  rebaseConflictState: RebaseConflictState | null
+) {
+  if (rebaseConflictState !== null) {
+    // there should be changes to rebase if conflicts are detected
```

Comment:
It's unclear to me how this comment relates to what's below. When I read this, I expected to see us look for files with a conflict marker, not files that are untracked.

Reply:
Let me take a stab at a better comment here. The code in here only really cares about these two cases:

 - all files are tracked by Git -> all files are needed to rebase -> "Include All" is true
 - some untracked files found -> some files are needed for rebase -> "Include All" is mixed state

There's actually a third state here, now I think about it:

 - no changed files in working directory -> rebase will skip the current commit -> "Include All" is false

This is relatively minor because the checkbox is disabled, but I'd like to ensure this reflects reality while we're in here making changes...

Reply:
Check out 0e2e5406b and see if that makes more sense